src/main/java/com/thirdcc/webapp/authorization/ManagementTeamSecurityExpression.java

```diff
@@ -39,12 +40,27 @@ public ManagementTeamSecurityExpression(
         this.yearSessionRepository = yearSessionRepository;
     }
 
+    /**
+     * Check if User is CC Head only
+     */
     public boolean isCurrentCCHead() {
         User currentUser = getCurrentUserWithLogin();
         YearSession currentYearSession = yearSessionRepository.findFirstByOrderByIdDesc()
             .orElseThrow(() -> new BadRequestException("Year Session not found"));
```

Comment:
suggest to put this into a YearSessionService, 
the caller no need to know the logic of how to get the 'current' year session.

In the future, if we want to change the logic to get the 'current' year session we can change that only in the service

Reply:
resolved in 24b680f2ed9587a7b644cb8051a6fe0babb5615e